project/Dependencies.scala

```diff
@@ -6,18 +6,18 @@ import Keys._
 object Dependencies {
   import DependencyHelpers._
 
-  val sparkVersion = "3.0.2"
+  val sparkVersion = "3.2.0"
   val scalaTestVersion = "3.0.8"
   val junitVersion = "5.8.2"
   val akkaVersion = "2.6.14"
   val akkaHttpVersion = "10.2.4"
   val springBootVersion = "2.6.2"
   lazy val logbackVersion = "1.2.3"
   lazy val loggingVersion = "3.9.0"
-  lazy val slf4jVersion = "1.7.25"
+  lazy val slf4jVersion = "1.7.30"
```

Comment:
the mleap-spark require slf4jVersion >= 1.7.30 if we upgrade spark to 3.2.0. because slf4jVersion new version add new methods which used by spark 3.2